src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -173,7 +176,10 @@ public override CodeGenOutput GenerateOutput(ServiceInfo service)
 						code.WriteLine($"return new {capModuleName}HttpClient(options);");
 
 					code.WriteLine();
-					code.WriteLine("const { fetchResponse, createResponseError, createRequiredRequestFieldError } = HttpClientUtility;");
+					var utilityImports = new List<string> { "fetchResponse", "createResponseError", "createRequiredRequestFieldError" };
+					if (httpServiceInfo.Events.Count > 0)
+						utilityImports.Add("createFetchEventStream");
+					code.WriteLine($"const {{ {string.Join(", ", utilityImports)} }} = HttpClientUtility;");
```

Comment:
Not actually suggesting a change, but it's funny that a raw string inverts what doubled brackets mean.
```suggestion
					code.WriteLine($$"""const { {{string.Join(", ", utilityImports)}} } = HttpClientUtility;""");
```